### PR TITLE
Restore --bare option. Resolves #1511

### LINKED ISSUE #1511: Citation rendering fails with bare repository (rugged adaptor)
Rendering citations in a bare repository stops working with commit f6f81b39, due to the switch to the rugged adaptor and removal of the --bare option.

### PR DESCRIPTION
**Pull request by dometto**

### CODE REVIEW

File: bin/gollum [hunk 1]
```diff
@@ -61,6 +61,9 @@ opts = OptionParser.new do |opts|
   opts.on('-r', '--ref [REF]', 'Specify the branch to serve. Default: \'master\'.') do |ref|
     wiki_options[:ref] = ref
   end
+  opts.on("--bare", "Declare '<git-repo>' to be bare. This is only necessary when using the grit adapter.") do
```
Comment (bartkamphorst) [inline bin/gollum]: @dometto Does #1511 show that `This is only necessary when using the grit adapter.` is false?

Comment (dometto) [reply bin/gollum]: It does. Dang, went too fast -- that notice needs to be removed.

Comment (dometto) [reply bin/gollum]: I've now fixed this on my local 5.x branch that will be merged into `master`.

Comment (bartkamphorst) [reply bin/gollum]: Excellent.

**Pull request merged**